tensorflow/lite/delegates/flex/build_def.bzl

```diff
@@ -141,6 +141,7 @@ def tflite_flex_cc_library(
                 clean_dep("//tensorflow/core:portable_tensorflow_lib_lite"),
                 clean_dep("//tensorflow/core/platform:strong_hash"),
                 clean_dep("//tensorflow/lite/delegates/flex:portable_images_lib"),
+                clean_dep("@com_google_protobuf//:protobuf"),
```

Comment:
I am fine with this too.

Reply:
And what happen if you don't add this?
Usually the protos_all_cc will implicitly pull in this dependency.

Reply:
Without this I get a runtime error when loading the tflite model in my app:

`java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "_ZNK6google8protobuf7Message11GetTypeNameEv" referenced by "/data/app/org.auderenow.openrdt.reader-2/lib/arm64/libtensorflowlite_flex_jni.so".`

It makes the aar file about 1 MB larger, so would definitely be interested to know if there is a lighter-weight fix

Reply:
I'll try to reproduce this. Not sure why GetTypeName is not pulled in.

Reply:
I tried it but cannot reproduce this issue.
The flex model run fine in my test.

When checking the libtensorflowlite_flex_jni.so with following command I get:
nm -gD libtensorflowlite_flex_jni.so | grep GetTypeName
 U _ZNK6google8protobuf7Message11GetTypeNameEv

Which means GetTypeName is included in the library.

Can you provide more details about your build environment? What tools are you using?

Reply:
Gentle ping on this.